test/cloudtest/pkg/commands/main.go

```diff
@@ -208,8 +208,15 @@ func PerformTesting(config *config.CloudTestConfig, factory k8s.ValidationFactor
 	// Fill tasks to be executed..
 	ctx.createTasks()
 
-	ctx.performExecution()
-	return ctx.generateJUnitReportFile()
+	err := ctx.performExecution()
+	reportfile, err2 := ctx.generateJUnitReportFile()
+	if err2 != nil && err != nil {
+		logrus.Errorf("Error during generation of report: %v", err2)
```

Comment:
Check if err2 not nil should be enough here

Reply:
Thanks.